src/myr/reference.js

```diff
@@ -284,6 +284,24 @@ let transformations = [
         description: <span>The setTexture function changes the texture of the cursor. The texture's normal color is displayed when setTextureColoring() is false, otherwise the texture will be affected by the current color. Textures can be applied either by using the name of one of <button id="textures" onClick = {handleTextureOpen} style = {{backgroundColor:"white", color:"#47a0ff", padding:0, border:"none"}}>these textures</button> or by inserting a valid url (e.g. "bricks" or "https://learnmyr.org/img/MYR-Logo.png"). WidthRepeat and heightRepeat will change how many times the texture is displayed on the object in each direction. If widthRepeat and heightRepeat are not declared they will be set to default settings. Depending on the policy of the website they’re from, image URLs may not be usable, which will result in a blank Texture and Color. An empty setTexture() or setTexture("") statement will remove the current texture.</span>,
         example: "setTexture"
     },
+    {
+        name: "setMaterialType",
+        parameters: [{type: "MaterialType", name: "materialtype"}],
+        description: <span>The setMaterialType function changes the material type of the cursor.</span>,
+        example: "setMaterialType(\"toon\");"
+    },
+    {
+        name: "setRoughness",
+        parameters: [{type: "number", name: "roughness"}],
+        description: <span>The setRoughness function changes the roughness of the cursor within a range of 0 to 1. Custom roughness only applies to objects with MaterialType.PHYSICAL.</span>,
+        example: "setRoughness(0.5)"
+    },
+    {
+        name: "setMetalness",
+        parameters: [{type: "number", name: "metalness"}],
+        description: <span>The setMetalness function changes the metalness of the cursor within a range of 0 to 1. Custom metalness only applies to objects with MaterialType.PHYSICAL.</span>,
+        example: "setMetalness(0.5)"
+    },
```

Comment:
"example" is used for a url link when user click on the example scene for the reference. So it should be just "setMaterialType" "setRoughness" and "setMetalness"